src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/CommPkg.cs

```diff
@@ -63,6 +65,7 @@ protected CommPkg()
         public int ProjectId { get; private set; }
         public bool RfocAccepted { get; set; }
         public ICollection<Certificate> CertificateScopes => _certificateScope;
+        public Guid Guid { get; set; } = Guid.Empty;
```

Comment:
remove init  Guid.Empty;
Add a Todo // make this private after FillGuids is completed